extensions/wikia/Insights/specials/UnconvertedInfoboxesPage.class.php

```diff
@@ -128,6 +128,18 @@ public function reallyDoQuery( $limit = false, $offset = false ) {
 	 * @return bool
 	 */
 	public static function isTitleWithNonportableInfobox( $titleText, $contentText ) {
+		// ignore docs pages
+		if ( strpos( $titleText, '/doc' ) ) {
+			return false;
+		}
+
+		$title = Title::newFromText( $titleText, NS_TEMPLATE );
+
+		// ingore redirects
+		if ( $title->isRedirect() ) {
```

Comment:
I'd suggest excluding this in the query (in `reallyDoQuery`) so we don't even iterate over redirects (i.e. `page_is_redirect = 0`).